operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -508,7 +508,8 @@ alertmanager:
   # -- Pod affinity settings for the alertmanager. This is ignored when alertmanager.zoneAwareReplication.enabled=true.
   affinity: {}
 
-  # -- topologySpreadConstraints allows to customize the default topologySpreadConstraints. This can be either a single dict as shown below or a slice of topologySpreadConstraints.
+  # -- topologySpreadConstraints allows to customize the default topologySpreadConstraints. This can be either a single dict as shown below or a slice of topologySpreadConstraints. 
+  # minDomains: optional field supported
```

Comment:
Can you move this under the `topologySpreadConstraints` key? So perhaps something like 
```yaml
 # labelSelector is taken from the constraint itself (if it exists) or is generated by the chart using the same selectors as for services.
  topologySpreadConstraints:
    maxSkew: 1
    topologyKey: kubernetes.io/hostname
    whenUnsatisfiable: ScheduleAnyway
    # minDomains: 1
```